handlers/graph.go

```diff
@@ -81,9 +81,15 @@ func graphNamespaces(o options.Options, client *prometheus.Client) graph.Traffic
 
 	trafficMap := graph.NewTrafficMap()
 	for _, namespace := range o.Namespaces {
+		// If we don't have access to the namespaces being requested then return an error.
```

Comment:
Don't do this check here, handle it in the options processing.  This code should be able to assume that the requested namespaces are valid and just focus on graph generation, not security concerns.